playground/dist/app.js

```diff
@@ -233,14 +233,17 @@ class MongoService {
     }
 
     if (isResponseOk) {
-      const { total, data: rset, types } = responseContent;
+      const { data: rset, types } = responseContent;
       updateLastExecutedQuery(query);
       let dataset = mongoResultsToDataset(rset);
-      dataset.paginationContext = {
-        totalCount: total,
-        pagesize: limit,
-        pageno: Math.floor(offset / limit) + 1,
-      };
+      const paginationInfo = rset.pagination_info;
+      const pageNumber = Math.floor(offset / limit) + 1;
+      const pageSize = limit;
+      dataset.paginationContext = getPaginationContext(
```

Comment:
By looking at the deployed version of this PR (https://weaverbird-playground-pr-1623.onrender.com/), looks like `getPaginationContext` is not defined.
If we want to use it here, we should add it to the `export`s in main.ts, and get it from the `vqb` variable (L2-15 of this file). (The `vqb` variables contains all the exports of the module)

Reply:
Sorry, it was super big but I didn't notice it 😆 🤦‍♀️